src/unit1/lesson7/README.md

```diff
@@ -137,11 +136,25 @@ namespace ContactsManager
             {
                 var store = new DocumentStore
                 {
-                    Urls = new [] {"http://localhost:8080"},
+                    Urls = new[] { "http://localhost:8080" },
                     Database = "ContactsManager"
                 };
 
-                return store.Initialize();
+                store.Initialize();
+
+                // Gets all database names existing in RavenDB Server
+                var databaseNames =
+                    store.Maintenance.Server.Send(new GetDatabaseNamesOperation(0, 500));
```

Comment:
Better to use `GetDatabaseRecordOperation`, instead, and check for `null`

Reply:
This suggestion was was implemented!!!
😃